pyccel/codegen/printing/pycode.py

```diff
@@ -418,6 +418,18 @@ def _print_NumpyAutoFill(self, expr):
                 dtype = dtype,
                 order = expr.order)
 
+    def _print_NumpyLinspace(self, expr):
+        return expr.__str__()
```

Comment:
You shouldn't do this for 2 reasons:
1. `expr.__str__()` should be written as `str(expr)`
2. `str(expr)` does not always give the same result as `_print(expr)`. Although the contents for `_print_NumpyLinspace` may be the same, that does not mean that they are also the same for `_print(expr.start)` for example